src/components/fields/MultipleChoice.js

```diff
@@ -145,41 +146,52 @@ class MultipleChoice extends AskField {
     return { options: selectedOptions }
   }
 
-  render () {
+  getCharIndex(i) {
+    return String.fromCharCode(65 + i);
+  }
+
+  render() {
     return (
       <div>
         <fieldset
-          style={styles.base}>
-          <legend style={styles.accesibleLegend}>{this.props.title}</legend>
-          {this.props.options && !!this.props.options.length
-            ? <div>
+          style={ styles.base }>
+          <legend style={ styles.accesibleLegend }>{ this.props.title }</legend>
+          { this.props.options && !!this.props.options.length ?
```

Comment:
Yes, that's the linter spec, looks better to me too, I'll do the final linting after merging.